include/addendum.txt

```diff
@@ -0,0 +1,73 @@
+.. _addendum:
+
+Addendum
+========
+
+Currently a large process is underway to change how the middleware operates.
+This new version is the Gen3 Middleware and consists of a rewrite of Butler as
+well as the addition of SuperTasks and pre-flight. The pre-flight will use the
+`Gen3 Butler`_, as well as the `SuperTask`_ to create a **quantum graph**
+(workflow like graph) that can be executed to run the actual pipelines. This
+work is in the implementation phase, depending on the results and changes of
+this Gen3 Middleware we will re-evaluate some of the workflow systems to see
+which on fits best in this framework.
```

Comment:
"which on fits best in this framework"   ->  which one fits best with this new middleware

I would even say "which one fits best with this new middleware and the project requirements"